README.md

```diff
@@ -134,6 +136,7 @@ Some of these optional requirements require work in the front-end only, back-end
 - Be able to check in videos from the customer details page
 - Full CRUD for customers
 - Introduce scrolling pagination to dynamically load more videos when scrolling to the bottom (research!!)
+- Create tests for each component using previous projects and the [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/) docs as a guide.
```

Comment:
Oh, we're having them do tests this time.

Reply:
This is in the optional section,

Reply:
Oh, duh.  🤦🏻‍♀️